src/CacheItemPoolInterface.php

```diff
@@ -89,7 +89,7 @@ public function clear();
      * @return bool
      *   True if the item was successfully removed. False if there was an error.
      */
-    public function deleteItem($key);
+    public function deleteItem(string $key): bool;
```

Comment:
would it make sense to allow the new `Stringable` type?

Reply:
It's a judgement call either way.  I went with `string|Stringable` only when the existing documentation implied an object was OK.  In this case, the key should be a trivial opaque string, not a formatted string like a URL or IP address (where __toString makes sense to use).  I cannot think of when using an object as the key ID would make sense, and even if someone is, tossing `(string)` into the function call is trivial for them to do.

Reply:
I've used the `__toString` in situations where computing the key is expensive, so `Stringable` was allowing me to have sort of "lazy strings", but thinking it twice, in this context `string` is good.